File: benchmarks/programs/fibonacci/fibonacci.js
const n = 30;
console.log(0);
console.log(1);
let a = 0,
  b = 1;
for (let i = 3; i <= n; i++) {
  const temp = a + b;
  console.log(temp);
  a = b;
  b = temp;
}

File: benchmarks/programs/fibonacci/fibonacci.c
#include <stdio.h>

int main() {
    int n = 30;
    printf("0\n");
    printf("1\n");
    int a = 0, b = 1;
    for (int i = 3; i <= n; i++) {
        int temp = a + b;
        printf("%d\n", temp);
        a = b;
        b = temp;
    }
    return 0;
}
